opba-auth-rest-api/src/main/resources/static/tpp_auth_api.yml

```diff
@@ -42,20 +42,39 @@ paths:
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/PsuAuthBody'
+              $ref: "#/components/schemas/PsuAuthBody"
```

Comment:
Double quotes process '\n' characters in string as if it would be new line, this is not what is desired in URL path references, please use `'` for quoting 
Please check quoting here and below